### PR TITLE
Fix Satan Morocc on Warper

### PR DESCRIPTION
**Pull request by mazvi**
Fix Satan Morocc on Warper

* **Addressed Issue(s)**: 

* **Server Mode**: 

* **Description of Pull Request**:

### CODE REVIEW

File: npc/custom/warper.txt [hunk 1]
```diff
@@ -257,7 +258,7 @@ F23: setarray @c[0],208,227,190,206,240,206,190,143,307,252,239,213,185,188,193,
 F24: Restrict("Pre-RE",2,7,9,10,11,13);
 	 setarray @c[2],192,162,235,166,202,206,202,208,225,202,202,214,263,196,217,201,87,121,277,181,221,185,175,200,174,197;
 	 Disp("Rachel Field",1,13); Pick("ra_fild");
-F25: if(.Satan_Morocc){
+F25: if(.Satan_Morocc=1){
```
Comment (Daegaladh) [inline npc/custom/warper.txt:261]: That makes no sense, it will set .Satan_Morocc to 1/true. Even with == will do exactly the same as before.

**Review by Daegaladh — CHANGES_REQUESTED** [review]

File: npc/custom/warper.txt (hunk 1 above)
Comment (Lemongrass3110) [inline npc/custom/warper.txt:261]: This is an assignment and no compare now.
Therefore this does not fix something, rather break something.

Comment (Lemongrass3110) [reply npc/custom/warper.txt:261]: Oh... Sorry github did not refresh it seems.
Totally agree with @Daegaladh here.

**Comment by mazvi** [comment]
I have tried it. if without =1 then the function doesn't work, try it.

**Comment by mazvi** [comment]
Sorry I was wrong, my function didn't work because I was using callfunc. :)

**Pull request closed without merging**